deploy/site.yaml

```diff
@@ -0,0 +1,22 @@
+domain: phone.docs.ubuntu.com
+
+image: prod-comms.docker-registry.canonical.com/phone.docs.ubuntu.com
+
+useProxy: false
```

Comment:
?

Reply:
You are right, it wasn't like this before but we are using it for docs.ubuntu.com:
https://github.com/canonical-web-and-design/docs.ubuntu.com/blob/master/deploy/site.yaml#L5

Should we remove it there? This property is to not use:
https://github.com/canonical-web-and-design/deployment-configs/blob/0bcdaf93f9754e0ab458ba4514eaf66d37e92102/templates/proxy-settings.yaml

Reply:
yeah I don't know why we allow this to be optional. It should be removed from both. I'm sure the docs sites, which are static, have no interaction with the proxy anyway.

Reply:
> I don't know why we allow this to be optional

:thinking:  looks like we are using it a lot:
https://github.com/search?p=1&q=org%3Acanonical-web-and-design+useProxy&type=Code

I am not sure now, but we did a deep review of these properties some time ago. Do you think it should be fine for all the site to have it by default?

Reply:
No, I have no confidence that *all* sites can safely use the proxy. But I do have high confidence that static sites can use the default. And we should *definitely* not be adding more sites with `useProxy: false`.

Reply:
Okay, I will update this property also for docs.u.c at the same time I do the redirection rules.